### PR TITLE
Add Woof token

### PR DESCRIPTION
**Pull request by atmoner**

### CODE REVIEW

**Review by hkxx — APPROVED** [review]

**Review by hkxx — APPROVED** [review]

**Review by hkxx — APPROVED** [review]
Ok good

**Review by hkxx — APPROVED** [review]

File: osmosis/assetlist.json [hunk 1]
```diff
@@ -1316,6 +1316,49 @@
...
+        },
+        {
+          "denom": "WOOF",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/9B8EC667B6DF55387DC0F3ACC4F187DA6921B0806ED35DE6B04DE96F5AB81F53",
+      "name": "WOOF chihuahua",
+      "display": "woof",
```
Comment (JeremyParish69) [inline osmosis/assetlist.json:1338]: this display denom is not defined within denom_units. Should be 'WOOF'?

Comment (atmoner) [reply osmosis/assetlist.json]: True! Fixed on this commit https://github.com/cosmos/chain-registry/pull/3545/commits/e6a428aa3cccb2826404068ec18b606218cb0ac9
Thanks

**Review by woof-chihuahua — APPROVED** [review]

**Review by El-Jay-Crypto — APPROVED** [review]
Woof!

**Pull request merged**